server/server.go

```diff
@@ -209,12 +209,13 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 	tlsConfig, err := util.LoadTLSCertificates(s.cfg.Security.SSLCA, s.cfg.Security.SSLKey, s.cfg.Security.SSLCert)
 	if err != nil {
 		logutil.BgLogger().Error("secure connection cert/key/ca load fail", zap.Error(err))
-		return nil, err
```

Comment:
why remove this line?

Reply:
after add require-secure-connection option, it's better follow mysql logic, server will stop if require-secure-connection = true, but we should not stop if user didn't want to require tls.